pkg/kubectl/cmd/label.go

```diff
@@ -57,6 +57,9 @@ $ kubectl label pods foo bar-`
 )
 
 func NewCmdLabel(f *cmdutil.Factory, out io.Writer) *cobra.Command {
+	p := kubectl.NewHumanReadablePrinter(false, false, false, false, []string{})
```

Comment:
Agreed, it should be added.  But this patch is correct for kube. If others want to build on it I certainly think they should. This was pointed out to OpenShift people back when support for `get` was added...